ports/libmlservo/src/lib.rs

```diff
@@ -109,150 +102,73 @@ pub unsafe extern "C" fn init_servo(
     app: MLApp,
     logger: MLLogger,
     history_update: MLHistoryUpdate,
+    url_update: MLURLUpdate,
     keyboard: MLKeyboard,
     url: *const c_char,
     width: u32,
     height: u32,
     hidpi: f32,
 ) -> *mut ServoInstance {
-    // Servo initialization goes here!
-    servo::embedder_traits::resources::set(Box::new(ResourceReaderInstance::new()));
     let _ = log::set_boxed_logger(Box::new(logger));
     log::set_max_level(LOG_LEVEL);
-    let gl = GlesFns::load_with(|symbol| {
-        let cstr = CString::new(symbol).expect("Failed to convert GL symbol to a char*");
-        eglGetProcAddress(cstr.as_ptr() as _) as _
-    });
 
+    let gl = gl_glue::egl::init().expect("EGL initialization failure");
     info!("OpenGL version {}", gl.get_string(gl::VERSION));
```

Comment:
I think we can get rid of this `info!`, it was useful while debugging but not any more.